recipe/meta.yaml

```diff
@@ -12,14 +12,16 @@ source:
     - conda-build.patch
 
 build:
-  number: 7
+  number: 0
   skip: true  # [not win]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
     - {{ compiler('m2w64_fortran') }}
+    - perl
+    - 7zip
```

Comment:
as there are patches, you will need to add m2-patch/patch

Reply:
Thanks! I've added `m2-patch`. Not adding `patch` at the moment since this is a Windows only package.